File: Fall-2019/Labs/Loop/8-3.go
/*
 *
 * In The Name of God
 *
 * +===============================================
 * | Author:        Parham Alvani <parham.alvani@gmail.com>
 * |
 * | Creation Date: 11-11-2019
 * |
 * | File Name:     8-3.go
 * +===============================================
 */

package main

import "fmt"

func main() {
	var n int

	fmt.Scanf("%d", &n)

	for i := 1; i <= n; i++ {
		for j := 0; j < i; j++ {
			fmt.Print("*")
		}
		fmt.Println()
	}
}

File: Fall-2019/Labs/Midterm/8-3.c
/*
 * In The Name of God
 * =======================================
 * [] File Name : 8-3.c
 *
 * [] Creation Date : 21-11-2019
 *
 * [] Created By : Parham Alvani <parham.alvani@gmail.com>
 * =======================================
*/
/*
* Copyright (c)  2019 Parham Alvani.
*/

#include <stdio.h>

int f();

int main() {
  for (int i = 0; i < 10; i++) {
    printf("%d: %d\n", i + 1, f());
  }
}

int f() {
  static int k = 0;
  k++;
  if (k % 2 == 0) {
    return 0;
  } else {
    return 1;
  }
}
